traitcuration/traits/templates/traits/trait_detail.html

```diff
@@ -76,47 +75,132 @@
+          class="suggestion-table__row 
+          {% if suggestion.term_id == current_term %} suggestion-table__row--current {% endif %}">
+            <td>
+              <span class="suggestion-table__link">
+                {{suggestion.term_id.label}} 
+                {% if suggestion.term_id == current_term %} 
+                <span class="uk-label label--primary">MAPPED TERM</span>
+                {% endif %}</span>
+            </td>
+            <td>{{suggestion.term_id.description}}</td>
+            <td> {{suggestion.term_id.cross_refs}}</td>
+            <td> {{suggestion.made_by.get_full_name}}</td>
+            <td>
+              <span class="uk-label label--{{status_dict | get_dict_item:suggestion.term_id.status}}">
+                {{suggestion.term_id.status | status_readable_name}}</span>
+            </td>
+          </tr>
+          {% endif %}
+        {% endfor %} {% endwith %} {% endwith %}
+        </tbody>
+      </table>
+      
+      {% if not user.is_anonymous %}
+      <button onclick="mapButtonClicked()" class="uk-button button button--primary">MAP TO SELECTION</button>
+      {% endif %}
+    </section>
 
+    <section class="section">
+      <h4> OTHER ACTIONS </h4>
+      <div class="actions">
+        <div class="action__container">
+          <span class="action__title">Suggest an existing term</span>
+          <input id="existingTermIRI" class="action__input" type="text" placeholder="Paste an existing term IRI...">
+          <button class="uk-button button button--primary" onclick="existingTermButtonClicked()">
+            ADD TERM AND MAP
+          </button>
+        </div>
+        <div class="action__container">
+          <span class="action__title">Suggest a new term</span>
+          <button class="uk-button button button--primary" onclick="newTermButtonClicked()">
+            CREATE NEW TERM
+          </button>
+        </div>
+      </div>
+      <div class="new_term hidden">
+        <h4>New Term Information</h4>
+        <form id="form" action="{% url 'add_mapping' trait.id %}" method="post">
+          {% csrf_token %}
+          <div class="fields">
+            <div class="field__container">
+              <label for="{{ new_term_form.label.id_for_label }}">Label:</label>
+              {{ new_term_form.label }}
+            </div>
+            <div class="field__container">
+              <label for="{{ new_term_form.description.id_for_label }}">Description:</label>
+              {{ new_term_form.description }}
+            </div>
+            <div class="field__container">
+              <label for="{{ new_term_form.cross_refs.id_for_label }}">Cross references:</label>
+              {{ new_term_form.cross_refs }}
+            </div>
+          </div>
+          <input type="submit" value="SUBMIT AND MAP" class="button button--primary uk-button new_term__button">
+        </form>
+      </div>
+    </section>
```

Comment:
Formatting of this section currently looks a bit cramped:
![image](https://user-images.githubusercontent.com/10669118/87938806-03bbe100-ca8f-11ea-967b-7c50cbd24898.png)

I would suggest adding more space between all elements. Especially the two section for the "existing" and "new" terms should be well separated.

Reply:
I am using the `gap` property to space those out, but it turns out that browser support for that is very limited so I will change it back to having `margin`